crates/sui-graphql-rpc/src/types/object.rs

```diff
@@ -566,6 +616,155 @@ impl Object {
 
         Ok(conn)
     }
+
+    async fn query_live(db: &Db, address: SuiAddress) -> Result<Option<Self>, Error> {
+        let vec_address = address.into_vec();
+        use objects::dsl as objects;
+
+        let stored_obj: Option<StoredObject> = db
+            .execute(move |conn| {
+                conn.result(move || {
+                    objects::objects
+                        .filter(objects::object_id.eq(vec_address.clone()))
+                        .limit(1)
+                        .into_boxed()
+                })
+                .optional()
```

Comment:
nit: can use `first` for this, and we no longer need the `into_boxed`!
```suggestion
                conn.first(move || {
                    objects::objects
                        .filter(objects::object_id.eq(vec_address.clone()))
                })
                .optional()
```